lib/ace/keyboard/textinput.js

```diff
@@ -201,6 +201,15 @@ var TextInput = function(parentNode, host) {
         if (afterContextMenu)
             afterContextMenu = false;
     };
+
+    var onKeyDown = function(e) {
+    	if(!inComposition && e.shiftKey && e.ctrlKey && host.session.$bidiHandler.hasRtlSupport()
```

Comment:
Should this be done on keyup instead, to not interfere with `ctrl-shift-` shortcuts?
Also note that some laptops (e.g. macbook) do not have right ctrl key, so this needs to be customizable.